crates/holochain/Cargo.toml

```diff
@@ -299,10 +299,19 @@ wasmer_sys = [
 wasmer_wamr = ["wasmer/wamr", "holochain_wasmer_host/wasmer_wamr"]
 
 # Enable chain head coordination
-chc = ["bytes", "reqwest", "holochain_conductor_api/chc", "holochain_chc/http"]
+chc = [
+  "bytes",
```

Comment:
The same with these please, when we touch them, can we migrate to using `dep:bytes` and `dep:reqwest`

Reply:
It avoids creating something that presents like a feature in auto-completion for libraries